src/api/service/axios-factory.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios'
+import {SecretManager, SVC_IDIR_SPEC} from './secret-manager'
+import {AxiosJiraClient} from './axios-jira-client'
+import {AxiosBitBucketClient} from './axios-bitbucket-client'
+
+export class AxiosFactory {
+  static async jira(): Promise<AxiosJiraClient> {
+    const entry = await (await SecretManager.getInstance()).getEntry(SVC_IDIR_SPEC)
+    const idirUsername = (await entry.getProperty(SVC_IDIR_SPEC.fields.UPN.name)).getPlainText()
+    const idirPassword = (await entry.getProperty(SVC_IDIR_SPEC.fields.PASSWORD.name))
+
+    return new AxiosJiraClient(axios.create({
+      baseURL: 'https://bwa.nrs.gov.bc.ca/int/jira',
```

Comment:
May need to add the environment here as testing should be done on the test instance of jira.

Reply:
Updated to read from environment variable and default to those values.